backend/FwLite/FwDataMiniLcmBridge/Api/FwDataMiniLcmApi.cs

```diff
@@ -818,7 +818,7 @@ public IAsyncEnumerable<Entry> SearchEntries(string query, QueryOptions? options
     {
         if (string.IsNullOrEmpty(query)) return null;
         return entry => entry.CitationForm.SearchValue(query) ||
-                        entry.LexemeFormOA.Form.SearchValue(query) ||
+                        entry.LexemeFormOA?.Form.SearchValue(query) == true ||
```

Comment:
`!!` is better than `== true` if there's any chance that SearchValue might ever return a string or other truthy but non-Boolean value.